fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -1713,6 +1726,7 @@ public boolean finishCloneTask(CloneTask cloneTask, TFinishTaskRequest request)
                 }
             } else if (e.getStatus() == Status.UNRECOVERABLE) {
                 // unrecoverable
+                Env.getCurrentSystemInfo().getBackend(tabletCtx.getDestBackendId()).updateCloneFailedWindow();
```

Comment:
delete line 1729.    unrecoverable is not really clone error,   it's cause by program bug or other error like  the clone tablet has been deleted like;

Reply:
line 1734.  should call  clearCloneFailedWindow()  // if clone one succ, clear the whole error windowns

Reply:
fixed